chapter_appendix-mathematics-for-deep-learning/integral-calculus.md

```diff
@@ -109,10 +109,10 @@ x = torch.arange(a, b, epsilon)
 f = x / (1 + x**2)
 
 approx = torch.sum(epsilon*f)
-true = torch.log(torch.tensor([2.])) / 2
+true = torch.log(torch.tensor([5.])) / 2
```

Comment:
Can you specify why changing to a "5" here?

Reply:
The integral of f = x / (1 + x**2) is log(1+x**2)/2, so it's 5 inside the log with x = 2.